src/Components/Notifications/__tests__/PartnerShowOpenedNotification.jest.tsx

```diff
@@ -34,12 +33,6 @@ const { renderWithRelay } = setupTestWrapperTL<
 })
 
 describe("PartnerShowOpenedNotification", () => {
-  beforeAll(() => {
-    ;(useFeatureFlag as jest.Mock).mockImplementation(
-      featureName => featureName === "onyx_new_notification_page"
-    )
-  })
-
   it("renders the notification", () => {
     renderWithRelay({
       Notification: () => notification,
```

Comment:
This spec is now failing randomly on main @olerichter00

Reply:
Thanks! This test uses a currently running show, which is now closed, and displays another text...

https://github.com/artsy/force/pull/13762/files